chore: update network names

juanmigdr opened the pull request:
<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been completely filled out, and PR status checks have passed at least once.
-->

## **Description**
This PR simplifies network names:
Linea -  ✅ 
Base - ✅ 
Arbitrum - ✅ 
Avalanche - ✅ 
BNB Chain - ✅ 
OP -  ✅ 
Polygon -  ✅ 
Sei - ✅ 
zkSync - ✅ 
Ethereum - ✅

<!--
Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/35727?quickstart=1)

## **Changelog**

<!--
If this PR is not End-User-Facing and should not show up in the CHANGELOG, you can choose to either:
1. Write `CHANGELOG entry: null`
2. Label with `no-changelog`

If this PR is End-User-Facing, please write a short User-Facing description in the past tense like:
`CHANGELOG entry: Added a new tab for users to see their NFTs`
`CHANGELOG entry: Fixed a bug that was causing some NFTs to flicker`

(This helps the Release Engineer do their job more quickly and accurately)
-->

CHANGELOG entry: update: simplified network names for better readability - "Ethereum Mainnet" to "Ethereum", "Linea Mainnet" to "Linea", "Base Mainnet" to "Base", "Arbitrum One" to "Arbitrum", "Avalanche Network C-Chain" to "Avalanche", "Binance Smart Chain" to "BNB Chain", "OP Mainnet" to "OP", "Polygon Mainnet" to "Polygon", "Sei Mainnet" to "Sei", and "zkSync Era Mainnet" to "zkSync Era" (#35734)

## **Related issues**

Fixes: https://consensyssoftware.atlassian.net/browse/ASSETS-1016

## **Manual testing steps**

New user flow:
1. Freshly install Metamask Extension and view the network names displayed (All of them should be like in the screenshot) to be merged

Existing user flow:
1. Update to the Metamask version from this PR
2. Look at all your network names after the update (Custom names should not have been updated)

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the before and after of your change. -->

### **Before**

<img width="500" height="882" alt="image" src="https://github.com/user-attachments/assets/96b7e036-d0fa-4e90-8b08-6e5c33f954da" />

### **After**

<img width="496" height="878" alt="image" src="https://github.com/user-attachments/assets/9f379f50-0958-4852-89ac-57fd781aa019" />

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

juanmigdr committed:
chore: update Avalanche network name

juanmigdr committed:
chore: update Polygon network name

juanmigdr committed:
chore: update Base network name

juanmigdr committed:
chore: update OP network name

juanmigdr committed:
fix: broken unit tests

juanmigdr committed:
chore: update BNB Chain network name

juanmigdr committed:
chore: update Linea network name

juanmigdr committed:
fix: broken test due to missing whitespace in snapshot

juanmigdr committed:
fix: missing Linea conversion

juanmigdr committed:
chore: update Arbitrum network name

juanmigdr committed:
chore: updated Sei network name

juanmigdr committed:
chore: added migrations to migrate existing users to new chain names

juanmigdr committed:
chore: created necessary tests

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
fix: solve linting issues

juanmigdr committed:
fix: failing pipelines unit tests and linting

juanmigdr committed:
fix: failing pipeline due to whitespace

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
chore: added new migrations

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
chore: remove unecessary changes

juanmigdr committed:
chore: updated snapshots

juanmigdr committed:
fix: failing tests

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
fix: broken your-accounts snapshot

juanmigdr committed:
fix: broken snapshot

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
fix: broken snapshot 2

juanmigdr committed:
fix: broken snapshots 3

juanmigdr committed:
fix: revert snapshot

juanmigdr committed:
fix: broken migration test

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
fix: failing snapshots

juanmigdr committed:
chore: updated zkSync network name

juanmigdr committed:
chore: added Changelog entry

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
fix: formatting issues changelog

juanmigdr committed:
chore: updated name change logic for new users

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
chore: use reusable names to avoid changing so many tests next time we update network names

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

Prithpal-Sooriya commented on test/e2e/tests/bridge/bridge-test-utils.ts:
```diff
@@ -977,7 +977,7 @@ export const getBridgeL2Fixtures = (
         '0x1': true, // Mainnet
         '0xe708': true, // Linea (source chain for test)
-        '0xa4b1': true, // Arbitrum One (destination chain for test)
+        '0xa4b1': true, // Arbitrum (destination chain for test)
```
since this is a comment change, lets revert to reduce file changes.

juanmigdr replied:
done

Prithpal-Sooriya commented on test/e2e/tests/solana/solana-networks.spec.ts:
```diff
@@ -41,7 +41,7 @@ describe('Solana network', function (this: Suite) {
         await accountMenu.switchToAccount('Account 1');
 
-        // Switch to Linea Mainnet to set it as the selected network
+        // Switch to Linea to set it as the selected network
```
only a comment change, lets revert to reduce PR size.

Prithpal-Sooriya commented on ui/components/component-library/avatar-network/README.mdx:
Storybook mdx, can be reverted to reduce PR size.

Prithpal-Sooriya commented on ui/components/component-library/picker-network/picker-network.stories.tsx:
Same for other stories. Can these be reverted to reduce PR size?

juanmigdr replied:
done

Prithpal-Sooriya commented on ui/components/multichain-accounts/multichain-site-cell/tool-tip/multichain-site-cell-tooltip.test.tsx:
```diff
@@ -66,7 +73,7 @@ describe('MultichainSiteCellTooltip', () => {
   const mockNetworks: EvmAndMultichainNetworkConfigurationsWithCaipChainId[] = [
     {
-      name: 'Ethereum Mainnet',
+      name: MAINNET_DISPLAY_NAME,
```
I like this cleanup!

HowardBraham committed:
test: a way to tolerate and log failure in E2E tests (#35927)

## **Description**

We needed a way to mark some E2E tests as essentially "optional" (it's
okay if they fail sometimes)

Because of the way tests are run, it was far easier to implement this as
looking to see if the file path includes `tolerate-failure`, rather than
writing a new option in `withFixtures`

## **Changelog**

CHANGELOG entry: null

## **Screenshots/Recordings**

The report still shows the test failure, but the workflow passes

<img width="843" height="383" alt="image"
src="https://github.com/user-attachments/assets/8d3ffd86-8690-46bf-8bfa-c05e42b99d68"
/>

Inside the results table

<img width="1036" height="50" alt="image"
src="https://github.com/user-attachments/assets/aa5a962b-0c1d-42c7-bf78-0c48386ccc86"
/>

<!--## **Related issues**
## **Manual testing steps**
## **Pre-merge author checklist**
## **Pre-merge reviewer checklist**-->

gauthierpetetin committed:
chore(branches): rename Version-vx.y.z branches into release/x.y.z (#35945)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

The purpose of this PR is to rename release branches from
`Version-vx.y.z` format into `release/x.y.z`, in order to align with
metamtask-mobile repo format.

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

[![Open in GitHub
Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/35945?quickstart=1)

## **Changelog**

<!--
If this PR is not End-User-Facing and should not show up in the
CHANGELOG, you can choose to either:
1. Write `CHANGELOG entry: null`
2. Label with `no-changelog`

If this PR is End-User-Facing, please write a short User-Facing
description in the past tense like:
`CHANGELOG entry: Added a new tab for users to see their NFTs`
`CHANGELOG entry: Fixed a bug that was causing some NFTs to flicker`

(This helps the Release Engineer do their job more quickly and
accurately)
-->

CHANGELOG entry: null

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask
Extension Coding
Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

juanmigdr commented on CHANGELOG.md:
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```
done

n3ps committed:
chore: rename maskicons to polycons cp-13.4.0 (#35955)

Renames maskicons to polycons

[![Open in GitHub
Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/35955?quickstart=1)

<!--
If this PR is not End-User-Facing and should not show up in the
CHANGELOG, you can choose to either:
1. Write `CHANGELOG entry: null`
2. Label with `no-changelog`

If this PR is End-User-Facing, please write a short User-Facing
description in the past tense like:
`CHANGELOG entry: Added a new tab for users to see their NFTs`
`CHANGELOG entry: Fixed a bug that was causing some NFTs to flicker`

(This helps the Release Engineer do their job more quickly and
accurately)
-->

CHANGELOG entry: null

Fixes:

1. click Menu > Settings
2. click General
3. scroll to Account icon

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

<!-- [screenshots/recordings] -->

<img width="380" height="159" alt="image"
src="https://github.com/user-attachments/assets/81f7a9dc-3a95-4c9e-a4f8-c1caa1818eaf"
/>

<!-- [screenshots/recordings] -->

- [x] I've followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask
Extension Coding
Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [x] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

n3ps committed:
fix: handle image load error (#35915)

## **Description**

Handle image load error on a dApp site's favicon

[![Open in GitHub
Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/35915?quickstart=1)

## **Changelog**

<!--
If this PR is not End-User-Facing and should not show up in the
CHANGELOG, you can choose to either:
1. Write `CHANGELOG entry: null`
2. Label with `no-changelog`

If this PR is End-User-Facing, please write a short User-Facing
description in the past tense like:
`CHANGELOG entry: Added a new tab for users to see their NFTs`
`CHANGELOG entry: Fixed a bug that was causing some NFTs to flicker`

(This helps the Release Engineer do their job more quickly and
accurately)
-->

CHANGELOG entry: null

## **Related issues**

Fixes: #35909 

## **Manual testing steps**

1. Connect to a site with a known broken favicon
2. It should fallback to first letter of that site

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<img width="114" height="76" alt="Screenshot 2025-09-15 at 10 33 05 AM"
src="https://github.com/user-attachments/assets/b90ba55e-102a-4f69-a21d-aa1aa34779d3"
/>



<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask
Extension Coding
Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [x] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

davidmurdoch committed:
feat: encourage users to update to a new version of the extension if a deeplink can be verified, but is not found (#35714)

## **Description**

When we release _new_ deep links old versions of MetaMask won't be able
to understand them, so they load a 404 page. To help remedy this problem
we now add a message to our 404 screen to suggest that the user should
update to the latest version.

We only add this message to "signed" links, as we can only verify that
signed links are real. User generated links could be junk.

In the future we'll check if there is an update available, and only show
the message if we know there is an update the user can update to.

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

[![Open in GitHub
Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/35714?quickstart=1)

## **Changelog**

<!--
If this PR is not End-User-Facing and should not show up in the
CHANGELOG, you can choose to either:
1. Write `CHANGELOG entry: null`
2. Label with `no-changelog`

If this PR is End-User-Facing, please write a short User-Facing
description in the past tense like:
`CHANGELOG entry: Added a new tab for users to see their NFTs`
`CHANGELOG entry: Fixed a bug that was causing some NFTs to flicker`

(This helps the Release Engineer do their job more quickly and
accurately)
-->

CHANGELOG entry: encourage users to update to a new version of the
extension if a deeplink can be verified, but is not found

<!--
## **Related issues**

Fixes:
-->

## **Manual testing steps**

To ensure the new message loads for signed URLs:

1. Go to
https://link.metamask.io/test?sig=rre9PwtZxvPVnTMVO16KxR3bA0rEIEuRXfN3w4tTDSf-GcJVZtXvJI6WA4KnjBB57b54FEVu7zS9hXArmDUZbg
2. Look at the screen. It should match the "after" below

To ensure the new message does NOT show for unsigned URLs:

1. Go to https://link.metamask.io/test
2. Look at the screen. It should match the "before" below

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<img width="468" height="744" alt="image"
src="https://github.com/user-attachments/assets/188ed5ef-552c-4a6a-95d7-d339e8ab56d2"
/>


### **After**

<img width="455" height="788" alt="image"
src="https://github.com/user-attachments/assets/c7c8fc50-97e8-4bdb-bcad-8bc4db62f6a9"
/>


## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask
Extension Coding
Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [x] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

lionellbriones committed:
feat: added ui for error states (#35874)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**
Added UI to display different error states on metamask shield membership
status


Figma:
https://www.figma.com/design/HTAO1SrmixV4ppv7qIvLoa/Metamask-Shield?node-id=10109-23294&t=DB7F42ij5tBF4dzz-4
<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

[![Open in GitHub
Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/35874?quickstart=1)

## **Changelog**

<!--
If this PR is not End-User-Facing and should not show up in the
CHANGELOG, you can choose to either:
1. Write `CHANGELOG entry: null`
2. Label with `no-changelog`

If this PR is End-User-Facing, please write a short User-Facing
description in the past tense like:
`CHANGELOG entry: Added a new tab for users to see their NFTs`
`CHANGELOG entry: Fixed a bug that was causing some NFTs to flicker`

(This helps the Release Engineer do their job more quickly and
accurately)
-->

CHANGELOG entry: Added UI for showing subscription error states

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->
<img width="623" height="821" alt="Screenshot 2025-09-16 at 11 24 06 AM"
src="https://github.com/user-attachments/assets/32fabde0-9d1b-4ed5-9d43-4df1ea79a912"
/>

<img width="456" height="188" alt="Screenshot 2025-09-16 at 11 24 12 AM"
src="https://github.com/user-attachments/assets/53a31e2c-4dc9-4a37-9fa7-14432a589a8f"
/>

<img width="623" height="835" alt="Screenshot 2025-09-16 at 11 25 44 AM"
src="https://github.com/user-attachments/assets/7874a87d-d7e0-4d83-99c7-f34f24b77bf1"
/>

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask
Extension Coding
Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

juanmigdr committed:
fix: solved some comments

juanmigdr committed:
fix: multiple comments

juanmigdr committed:
fix: reverted changes due to some comments

juanmigdr commented on shared/constants/swaps.ts:
```diff
@@ -137,7 +137,7 @@ export const SEI_SWAPS_TOKEN_OBJECT: SwapsTokenObject = {
   address: DEFAULT_TOKEN_ADDRESS,
   decimals: 18,
-  // SEI using the same icon as Sei Mainnet
+  // SEI using the same icon as Sei
```
done

juanmigdr commented on ui/components/multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal-network.stories.tsx:
done

juanmigdr committed:
fix: more comments

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
chore: revert removal white-spaces

juanmigdr commented on app/scripts/metamask-controller.js:
```diff
@@ -644,6 +644,12 @@ export default class MetamaskController extends EventEmitter {
+      // receive the correct, simplified "Popular" network names from the start.
+      networks[CHAIN_IDS.LINEA_MAINNET].name = 'Linea';
+      networks[CHAIN_IDS.BASE].name = 'Base';
+
```
There is no issue here since the only popular chains returned from Core are Linea, Base and Ethereum. Ethereum will be done in a separate MR

juanmigdr committed:
chore: updated migration to be 177

juanmigdr committed:
chore: minor cleanup removal of unecesary change

Prithpal-Sooriya reviewed (dismissed):
Nice work!

georgewrmarshall reviewed (commented):
Hello @juanmigdr, not sure if we(@MetaMask/design-system-engineers) are required for review for this one but it looks great! Are we doing the same on mobile and portfolio?

seaona reviewed (dismissed):
Changes on the e2e files LGTM

juanmigdr committed:
chore: updated Ethereum network name (#35947)

<!--
Please submit this PR as a draft initially.
Do not mark it as "Ready for review" until the template has been
completely filled out, and PR status checks have passed at least once.
-->

## **Description**

<!--
Write a short description of the changes included in this pull request,
also include relevant motivation and context. Have in mind the following
questions:
1. What is the reason for the change?
2. What is the improvement/solution?
-->

[![Open in GitHub
Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/35947?quickstart=1)

## **Changelog**

<!--
If this PR is not End-User-Facing and should not show up in the
CHANGELOG, you can choose to either:
1. Write `CHANGELOG entry: null`
2. Label with `no-changelog`

If this PR is End-User-Facing, please write a short User-Facing
description in the past tense like:
`CHANGELOG entry: Added a new tab for users to see their NFTs`
`CHANGELOG entry: Fixed a bug that was causing some NFTs to flicker`

(This helps the Release Engineer do their job more quickly and
accurately)
-->

CHANGELOG entry: updated network name from "Ethereum Mainnet" to
"Ethereum"

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

<!-- If applicable, add screenshots and/or recordings to visualize the
before and after of your change. -->

### **Before**

<!-- [screenshots/recordings] -->

### **After**

<!-- [screenshots/recordings] -->

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor
Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask
Extension Coding
Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format
if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling
guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)).
Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the
app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described
in the ticket it closes and includes the necessary testing evidence such
as recordings and or screenshots.

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

juanmigdr committed:
chore: fix failing test

juanmigdr commented:
> Hello @juanmigdr, not sure if we(@MetaMask/design-system-engineers) are required for review for this one but it looks great! Are we doing the same on mobile and portfolio?

Hey! We will be doing the same for mobile too, not idea about portfolio, maybe @alfeng6 has more insight on it

juanmigdr committed:
Merge branch 'main' into chore/1016-update-network-names-2

seaona reviewed (approved):
QA files look good! Left 1 question, but non-blocking

juanmigdr commented on test/e2e/tests/connections/multiple-provider-connections.spec.ts:
```diff
@@ -464,7 +480,7 @@ describe('Multiple Standard Dapp Connections', function () {
           driver,
           sitePermissionPage,
-          ['Ethereum Mainnet', 'Solana'],
+          [MAINNET_DISPLAY_NAME, 'Solana'],
```
We might want to also have it in the future, I just used the existing ones to avoid having to change those places in the future!

montelaidev reviewed (approved):
Accounts files looks good.

ccharly reviewed (approved):
LGTM for accounts-owned files.

cryptodev-2s reviewed (approved):
LGTM!
